### ISSUE TITLE
libretro.* marked as broken

### ISSUE THREAD

**Issue by aanderse** [NixOS/nixpkgs#102203]
**Describe the bug**

https://github.com/NixOS/nixpkgs/pull/101736 was a little bit too aggressive on the `libretro.*` package set. This PR set the entire package set as broken, even though only a few packages were.

**To Reproduce**
1. Add this to your `environment.systemPackages`: `(retroarch.override { cores = with libretro; [ beetle-snes mgba mupen64plus parallel-n64 snes9x snes9x2010 yabause ]; })` - note your configuration won't evaluate (unless you have set `allowBroken = true;`)

**Expected behavior**

Many of the packages in `libretro.*` aren't actually broken, so we should mark the broken packages individually instead of every package under `libretro.*`. Specifically [this line](https://github.com/NixOS/nixpkgs/blob/edb26126d98bc696f4f3e206583faa65d3d6e818/pkgs/misc/emulators/retroarch/cores.nix#L52) is too aggressive.

**Screenshots**

**Additional context**

**Notify maintainers**

@edwtjo @hrdinka @MP2E any motivation to tackle this?

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: libretro
# a list of nixos modules affected by the problem
module:
```

**Comment by veprbl** [comment]
This was fixed by #102145

**Comment by aanderse** [comment]
Oop! Thanks ❤️
